documentation/src/bootstrap_node.md

```diff
@@ -0,0 +1,93 @@
+# Bootstrap node
+
+## Introduction
+
+The bootstrap node is the first node a new node contacts when it joins the
```

Comment:
nit: "A bootstrap node is the first node a new node contacts when it joins the
network. It is responsible for providing new nodes with a list of other nodes
within that network, which can then be contacted to join said network. Every
Forest node has a list of bootstrap nodes that it can contact. This list is hardcoded into the node but can be modified by the user via the configuration file."